vendor/symfony/lib/plugins/sfPropelPlugin/lib/addon/sfPropelDatabaseSchema.class.php

```diff
@@ -604,7 +604,7 @@ protected function fixYAMLI18n()
           $this->setIfNotSet($this->database[$i18n_table], 'culture', array(
             'isCulture'  => true,
             'type'       => 'varchar',
-            'size'       => '7',
+            'size'       => 16,
```

Comment:
Don't really know why or if it matters but this size was set as a string (with quotes).

Reply:
Yeah, one was set to an int an one was set to a string so I went with int. Seems to work.